plugins/hexbin/filters/HexBin.cpp

```diff
@@ -192,7 +192,6 @@ void HexBin::done(PointTableRef table)
     {
         // Compute a UTM polygon
         BOX3D box = p.bounds();
-        int zone = SpatialReference::calculateZone(box.minx, box.miny);
```

Comment:
Was there something wrong here? Whether this line comes before or after the definition of the function would seem to make no difference.

Reply:
No there wasn't anything wrong. cppcheck has false positives in its analysis (it is not a full blown C++ compiler). That's a workaround to avoid one